src/app/shared/shared.module.ts

```diff
@@ -0,0 +1,12 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BreadcrumbsModule } from './components/breadcrumbs/breadcrumbs.module';
+import { FooterModule } from './components/footer/footer.module';
+import { HeaderModule } from './components/header/header.module';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule, HeaderModule, FooterModule, BreadcrumbsModule],
+  exports: [HeaderModule, FooterModule, BreadcrumbsModule],
+})
+export class SharedModule {}
```

Comment:
Personally, I don't like the idea to have a shared module in the application. In reality, you just need shared folder with index.ts file inside (which has all exports). Because you're currently following the rule 'module per component' you no longer require SharedModule in your application. Just import the required module per demand in that module, where you actually use it (in HTML template component, etc.).

Reply:
![shared-1](https://user-images.githubusercontent.com/12629525/166466588-4021e20e-e0ba-4be7-825e-651837a363fb.png)